### ISSUE TITLE
Can't select value from specific point

### ISSUE THREAD

**Issue by akloeber** [influxdb/influxdb#1309]
I'm not able to get the value of a specific point:

    curl -G 'http://localhost:8086/db/test/series?u=root&p=root&time_precision=u' --data-urlencode 'q=select * from ts limit 1'
    [{"name":"ts","columns":["time","sequence_number","value"],"points":[[1420981200837000,90001,9648.5]]}]

If I query exactly this point as described in http://influxdb.com/docs/v0.8/api/query_language.html#selecting-a-specific-point the attached value `9648.5` is not returned:

    curl -G 'http://localhost:8086/db/test/series?u=root&p=root&time_precision=u' --data-urlencode 'q=select * from ts where time = 1420981200837000 and sequence_number = 90001'
    [{"name":"ts","columns":["time","sequence_number"],"points":[[1420981200837,90001]]}]

Even if `value` is explicitely queried there is no difference in the result:

    curl -G 'http://localhost:8086/db/test/series?u=root&p=root&time_precision=u' --data-urlencode 'q=select value from ts where time = 1420981200837000 and sequence_number = 90001'
    [{"name":"ts","columns":["time","sequence_number"],"points":[[1420981200837,90001]]}]

In contrast to the first result the time values are not in microsecond epoch although `time_precision=u` is given.

Furthermore I always get a result even I use a timestamp and sequence number that is not present:

    curl -G 'http://localhost:8086/db/test/series?u=root&p=root&time_precision=u' --data-urlencode 'q=select * from ts where time = 9999999999 and sequence_number = 12345'
    [{"name":"ts","columns":["time","sequence_number"],"points":[[9999999,12345]]}]

Unfortunately I'm not able to reproduce that from scratch because of #1290 which causes that specific points are not returned at all.

**Environment:**
InfluxDB v0.8.3
Ubuntu 14.04.1 LTS

**Comment by joel-airspring** [comment]
I believe it will work if you add the precision 'u' to the timestamp in the db query itself, like
`q=select * from ts where time = 1420981200837000u and sequence_number = 90001`

The 'time_precision' query parameter in the url doesn't appear to control the precision of the timestamp in the db query.  Instead, it controls the precision of the returned data.

By the way, your link is to the 0.7 version of the docs, although they're not materially different from their 0.8 brethren.

**Comment by akloeber** [comment]
I checked again and if suffix `u` is appended the point is returned correctly but the example in the documentation should be updated accordingly and at http://influxdb.com/docs/v0.8/api/query_language.html#absolute-time it is said that the value in the query defaults to microsecond epoch and the suffix is only necessary if that should be changed.

Furthermore I still do not understand the result of the following query from above:

    curl -G 'http://localhost:8086/db/test/series?u=root&p=root&time_precision=u' --data-urlencode 'q=select * from ts where time = 9999999999 and sequence_number = 12345'
    [{"name":"ts","columns":["time","sequence_number"],"points":[[9999999,12345]]}]

It somehow looks like a point is found but the value is missing. Even if `time_precision=u` only controls the returned data it is unclear to me why 9999999 gets returned (should be 9999999999 or maybe 9999999999000).
In my opinion the logical answer would be `[]` (empty array).

PS: I updated the link to the most recent API documentation.
